source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -458,6 +458,18 @@ def __init__(self):
             hidden=True,
             help="Here, Max number of matrix points for the street network computation (limited to 100)",
         )
+        parser_get.add_argument(
+            '_sn_exclusion_area[]',
+            type=six.text_type,
+            case_sensitive=False,
+            hidden=True,
+            action='append',
+            dest='_sn_exclusion_areas',
+            help='Give 2 coords for an exclusion box. The format is like that:\n'
+            'Coord_1!Coord_2 with Coord=lat;lon\n'
+            ' - exemple : _sn_exclusion_area[]=2.40553;48.84866!2.41453;48.85677\n'
+            ' - This is a list, you can add to the maximun 20 _sn_exclusion_area[]\n',
```

Comment:
The max of 20 exclusion areas is not checked in the code. What happens if more than 20 are given ? 
Maybe keep only the first 20 to give to Here, and log a message in jormun ?

Reply:
Good point, I can protect and inform